src/emm/emm_cli.py

```diff
@@ -220,6 +231,22 @@ def list_modules(ctx: click.Context, enabled: bool, show_details: bool) -> None:
 
 
 @cli.command(context_settings=dict(max_content_width=100))
+@click.option("-r", "--revision", required=True, help="Revision to be checked out.")
```

Comment:
Since the revision is not just going to be used for "checkout", I think the help here needs to be reworded.